persist/fs/read.go

```diff
@@ -188,19 +310,27 @@ func (r *reader) readDigest() error {
 	if r.expectedIndexDigest, err = r.digestFdWithDigestContents.ReadDigest(); err != nil {
 		return err
 	}
+	if _, err := r.digestFdWithDigestContents.ReadDigest(); err != nil {
+		// Skip the summaries digest
```

Comment:
How come?

Reply:
We're not using these files yet, that will come in a later PR to keep this one a reasonable size.

Reply:
Cool makes sense.